applications/nrf_desktop/src/modules/Kconfig.usb_state

```diff
@@ -10,10 +10,20 @@ menu "USB state"
 
 config DESKTOP_USB_ENABLE
 	bool "Enable USB module"
-	depends on USB_DEVICE_HID
 	help
 	  Enables USB module.
 
+config DESKTOP_USB_SELECTIVE_HID_SUBSCRIPTION
```

Comment:
I think it will be simpler to just define usb_state_def.h with tables filled with ones. Now we have to track two places.

Reply:
I prefer to make this a feature enabled in KConfig. Maybe we should even add another Kconfig option to specify path to *.def file. Given board may in fact support various range of HID reports in different configurations and in that case customer may need to specify multiple usb_state.def files.

Reply:
The configurations will be backwards compatible too.

Reply:
At the moment we have two sources of configs - Kconfig and _def files. We don't specify location of _def files in Kconifg for any of our modules. If we introduce it this will have to go for every module.

But this is true - DK can work as a dongle and peripheral in which case we need to have separate configs.

Ok - let's keep it. I will add separate remarks.

Reply:
s/DESKTOP_USB_SELECTIVE_HID_SUBSCRIPTION/DESKTOP_USB_SELECTIVE_REPORT_SUBSCRIPTION

Reply:
done